src/materials/NEMLStress.C

```diff
@@ -21,17 +21,163 @@ InputParameters
 NEMLStress::validParams()
 {
   InputParameters params = NEMLStressBase::validParams();
+  params.addClassDescription("Use a constitutive model in the NEML library to compute stress");
   params.addRequiredParam<FileName>("database", "Path to NEML XML database.");
-  params.addRequiredParam<std::string>("model", "Model name in NEML database.");
+  params.addParam<std::vector<std::string>>("neml_variable_iname",
+                                            "Names of NEML xml name/value pairs");
+
+  params.addParam<Real>("neml_variable_value0",
+                        "NEML xml variable value for neml_variable_iname[0]");
+  params.addParam<Real>("neml_variable_value1",
+                        "NEML xml variable value for neml_variable_iname[1]");
+  params.addParam<Real>("neml_variable_value2",
+                        "NEML xml variable value for neml_variable_iname[2]");
+  params.addParam<Real>("neml_variable_value3",
+                        "NEML xml variable value for neml_variable_iname[3]");
+  params.addParam<Real>("neml_variable_value4",
+                        "NEML xml variable value for neml_variable_iname[4]");
+  params.addParam<Real>("neml_variable_value5",
+                        "NEML xml variable value for neml_variable_iname[5]");
+  params.addParam<Real>("neml_variable_value6",
+                        "NEML xml variable value for neml_variable_iname[6]");
+  params.addParam<Real>("neml_variable_value7",
+                        "NEML xml variable value for neml_variable_iname[7]");
+
+  params.addDeprecatedParam<std::string>(
+      "model",
```

Comment:
Can we not read files with multiple models defined anymore?

Reply:
No.  Is it important?  The xml interface to neml allowed that but the string interface doesn't and I switched made it so there is only a string interface.

Reply:
This is fixed with the new neml string interface update.  Our code now passes a model name too.